apps/meteor/server/settings/accounts.ts

```diff
@@ -743,6 +743,11 @@ export const createAccountSettings = () =>
 				i18nLabel: 'Sidebar_Sections_Order',
 				i18nDescription: 'Sidebar_Sections_Order_Description',
 			});
+
+			await this.add('Accounts_Default_User_Preferences_featuresPreview', '[]', {
```

Comment:
I think we have a "multiselect" type of setting that allows you to store an array of elements. That would be better suitable than this string